coverage.md

```diff
@@ -66,3 +66,4 @@ This file contains list of features and their test coverage.
 | As a user, I can export RPM-repository content to be consumed by a downstream Pulp3 instance. | PART | basic pulp-export succeeds |
 | As a user, I can import RPM-repository content exported from an upstream Pulp3 instance. | PART | basic pulp-import succeeds |
 | As a user, I can export/import an RPM repository with a kickstart | PART | Need to use rpm-distribution-tree fixture and check content after import |
+| As a user, I can use RBAC | PART | more testing is welcome |
```

Comment:
This needs a little more detail about what is or isn't tested (whichever is less)